packages/smooth_app/lib/background/background_task_details.dart

```diff
@@ -60,11 +62,31 @@ class BackgroundTaskDetails extends AbstractBackgroundTask {
     return null;
   }
 
+  /// Adds the background task and the pending changes.
```

Comment:
Could add some description for the return type?

From an external point of view, having an `addTask` and `_addTask` is a bit confusing.

Reply:
Fair enough.
The comment will be `Returns true if successful.`
Having an `addTask` and `_addTask` is indeed a bit confusing, but each method adds a task, and only the not-private one is erm... public.

Reply:
@M123-dev Fair enough.
Modified comment:
```dart
/// Returns true if successful (task added, local change added).
```